frontend/src/scenes/insights/insightDataLogic.ts

```diff
@@ -40,7 +40,10 @@ export const insightDataLogic = kea<insightDataLogicType>([
         values: [
             insightLogic,
             ['filters', 'insight', 'savedInsight'],
-            dataNodeLogic({ key: insightVizDataNodeKey(props) } as DataNodeLogicProps),
+            dataNodeLogic({
+                key: insightVizDataNodeKey(props),
+                loadPriority: props.loadPriority,
+            } as DataNodeLogicProps),
```

Comment:
What about adding this to the `DataNodeLogicProps` and `InsightLogicProps` types as well?

Reply:
DataNoteLogicProps already has a loadPriority, it was use by web analytics just not in these dashboards